hawkbit-artifact-repository-filesystem/src/test/java/org/eclipse/hawkbit/artifact/repository/ArtifactFilesystemRepositoryTest.java

```diff
@@ -61,16 +62,26 @@ public void getStoredArtifactBasedOnSHA1Hash() {
     public void deleteStoredArtifactBySHA1Hash() {
         final ArtifactFilesystem artifact = storeRandomArtifact(randomBytes());
 
-        artifactFilesystemRepository.deleteBySha1(artifact.getHashes().getSha1());
+        artifactFilesystemRepository.deleteBySha1(TENANT, artifact.getHashes().getSha1());
 
-        assertThat(artifactFilesystemRepository.getArtifactBySha1(artifact.getHashes().getSha1())).isNull();
+        assertThat(artifactFilesystemRepository.getArtifactBySha1(TENANT, artifact.getHashes().getSha1())).isNull();
+    }
+
+    @Test
+    @Description("Verfies that all artifacts of a tenant can be deleted in the file-system repository")
+    public void deleteStoredArtifactOftenant() {
+        final ArtifactFilesystem artifact = storeRandomArtifact(randomBytes());
+
+        artifactFilesystemRepository.deleteByTenant(TENANT);
+
+        assertThat(artifactFilesystemRepository.getArtifactBySha1(TENANT, artifact.getHashes().getSha1())).isNull();
     }
 
     @Test
     @Description("Verfies that an artifact which does not exists is deleted quietly in the file-system repository")
     public void deleteArtifactWhichDoesNotExistsBySHA1HashWithoutException() {
         try {
-            artifactFilesystemRepository.deleteBySha1("sha1HashWhichDoesNotExists");
+            artifactFilesystemRepository.deleteBySha1(TENANT, "sha1HashWhichDoesNotExists");
```

Comment:
May I ask you to add a test with a none existing tenant as well. Shoudn't be a problem but better safe than sorry